transactions/transactor_test.go

```diff
@@ -223,31 +223,8 @@ func (s *TxQueueTestSuite) TestCompleteTransaction() {
 			}
 			s.setupTransactionPoolAPI(args, testNonce, testNonce, selectedAccount, nil, testCase.signTxArgs)
 
-			w := make(chan struct{})
-			var sendHash gethcommon.Hash
-			go func() {
-				var sendErr error
-				sendHash, sendErr = s.manager.SendTransaction(context.Background(), args)
-				s.NoError(sendErr)
-				close(w)
-			}()
-
-			for i := 10; i > 0; i-- {
-				if s.manager.pendingSignRequests.Count() > 0 {
-					break
-				}
-				time.Sleep(time.Millisecond)
-			}
-
-			req := s.manager.pendingSignRequests.First()
-			s.NotNil(req)
-			approveResult := s.manager.pendingSignRequests.Approve(req.ID, "", testCase.signTxArgs, simpleVerifyFunc(selectedAccount))
-			s.NoError(approveResult.Error)
-			s.NoError(WaitClosed(w, time.Second))
-
-			// Transaction should be already removed from the queue.
-			s.False(s.manager.pendingSignRequests.Has(req.ID))
-			s.Equal(sendHash.Bytes(), approveResult.Response.Bytes())
```

Comment:
Here the test was checking that the `sendHash` received when creating the transaction request in the old version of `SendTransaction` matched the response of the result of the request. Now that `SendTransaction` returns the result itself, and skips the sendHash, what would be a more sensible thing to be checking for? Can we include the expected result in each testcase? How would we figure out what that result should be?

Reply:
we don't know which hash the function will return in advance, so regular "this is not zeroes" check should suffice